CHANGELOG.md

```diff
@@ -265,7 +265,7 @@ Removed 44 individual arrow ligatures, replaced with four universal, combinative
 
 #### 1.102
 
-- Support for IntelliJ-based IDEs ([instructions](https://github.com/tonsky/FiraCode/wiki/Intellij-products-instructions))
+- Support for IntelliJ-based IDEs ([instructions](https://github.com/tonsky/FiraCode/wiki/IntelliJ-products-instructions))
```

Comment:
GitHub appears to be case tolerant, but as it happens the case I'm using is the case used by the wiki, it's accidental that the other case is tolerated.